presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -2003,7 +2002,7 @@ private Session pushdownFilterAndNestedColumnFilterEnabled()
     {
         return Session.builder(getQueryRunner().getDefaultSession())
                 .setCatalogSessionProperty(HIVE_CATALOG, PUSHDOWN_FILTER_ENABLED, "true")
-                .setCatalogSessionProperty(HIVE_CATALOG, RANGE_FILTERS_ON_SUBSCRIPTS_ENABLED, "true")
+                .setCatalogSessionProperty(HIVE_CATALOG, "range_filters_on_subscripts_enabled", "true")
```

Comment:
Is there a reason we made this change?

Reply:
@imjalpreet This session property was not made private in Common since this session constant was getting used in Hive module. So I had used it directly for Test and made it private in common.

@yingsu00 You mentioned to keep all constants in `HiveCommonSessionProperties` as private. Can you suggest if we should maintain these properties constant list in `TestHiveUtil` or make them public in `HiveCommonSessionProperties` to use in Hive tests classes?

Reply:
Also, to add more context, many other such properties are public in HiveSessionProperties since the constants defined there are being used elsewhere. So, whichever way we decide to go here, we can do similar refactoring later for other properties as well.

Reply:
As discussed, updated `RANGE_FILTERS_ON_SUBSCRIPTS_ENABLED` as public in HiveCommonSessionProperties